logdb/logdb_test.go

```diff
@@ -19,12 +19,21 @@ import (
 	"github.com/vechain/thor/v2/tx"
 )
 
-func newTx() *tx.Transaction {
-	tx := new(tx.Builder).Build()
+func newTx(txType int) *tx.Transaction {
+	var trx *tx.Transaction
+	switch txType {
+	case tx.LegacyTxType:
+		trx = new(tx.LegacyBuilder).Build()
+	case tx.DynamicFeeTxType:
+		trx = new(tx.DynFeeBuilder).Build()
+	default:
+		panic(tx.ErrInvalidTxType)
```

Comment:
can we have a test for that too please

Reply:
I am just thinking, that it would be good to check if it triggers the panic when we have some invalid tx type (just to play safe :D )